src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -0,0 +1,184 @@
+	// Calculate deltas
+	upsert := make(map[string]interface{})
+	remove := make([]string, 0)
+	for k, v := range readbackConfig {
+		oldVal, haveOldVal := oldConfig[k]
+		if meta, ok := schema[k]; ok {
+			// For numeric types need special handling because
+			// yaml encoding will see '1' as an integer, even
+			// if it is given as the value for a floating point
+			// ('number') config property, and vice versa.
+			if meta.Type == "integer" {
+				switch vFloat := v.(type) {
+				case float64:
+					v = int(vFloat)
+				}
+
+				if oldVal != nil {
+					oldVal = int(oldVal.(float64))
+				}
+			} else if meta.Type == "number" {
+				switch x := v.(type) {
+				case int:
+					v = float64(x)
+				}
+			} else if meta.Type == "array" && meta.Items.Type == "string" {
+				if v != nil {
+					v = loadStringArray(v.([]interface{}))
+				}
+				if oldVal != nil {
+					oldVal = loadStringArray(oldVal.([]interface{}))
+				}
+			}
+
+			// For types that aren't numberic or array, pass them through as-is
+		}
+
+		if haveOldVal {
+			// If value changed, add it to list of updates
+			// DeepEqual because values can be slices
+			if !reflect.DeepEqual(oldVal, v) {
+				propertyDeltas = append(propertyDeltas, propertyDelta{k, fmt.Sprintf("%v", oldVal), fmt.Sprintf("%v", v)})
+				upsert[k] = v
+			}
+		} else {
+			// Present in input but not original config, insert
+			upsert[k] = v
+			propertyDeltas = append(propertyDeltas, propertyDelta{k, "", fmt.Sprintf("%v", v)})
+		}
+	}
+
+	for k := range oldConfig {
+		if _, found := readbackConfig[k]; !found {
+			meta, inSchema := schema[k]
+			if !inSchema {
+				continue
+			}
+
+			if !all && meta.Visibility == "tunable" {
+				continue
+			}
+			oldValue := oldConfig[k]
+			propertyDeltas = append(propertyDeltas, propertyDelta{k, fmt.Sprintf("%v", oldValue), ""})
+			remove = append(remove, k)
+		}
+	}
+
+	if len(upsert) == 0 && len(remove) == 0 {
+		fmt.Printf("No changes were made.\n")
+		return nil
+	}
+
+	tw := out.NewTable("PROPERTY", "PRIOR", "NEW")
+	for _, pd := range propertyDeltas {
+		tw.PrintStructFields(pd)
+	}
+	tw.Flush()
+
+	// Newline between table and result of write
+	fmt.Printf("\n")
+
+	// PUT to admin API
+	result, err := client.PatchClusterConfig(upsert, remove)
+	out.MaybeDie(err, "Error setting config: %v", err)
+	fmt.Printf("Successfully updated config, new config version %d.\n", result.ConfigVersion)
+
+	return nil
+}
+
+func newImportCommand(fs afero.Fs, all *bool) *cobra.Command {
+	var filename string
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Import cluster configuration from a file",
+		Long: `Import cluster-wide configuration properties
+
+Import configuration from a YAML file, usually generated with
+corresponding 'export' command.  This downloads the current cluster
+configuration, calculates the difference with the YAML file, and
+updates any properties that were changed.  If a property is removed
+from the YAML file, it is reset to its default value.  `,
```

Comment:
This help text looks quite narrow -- if it can be wrapped to 80col that'd be more consistent.

Reply:
As above, this is what `gwap` does (it indents the block, wraps to 80 chars, then you end up with shorter lines once it's unindented).